Deserts/Kuchen/Nusslebkuchen.md

```diff
@@ -0,0 +1,32 @@
+# Nusslebkuchen
+
+Dies ist das Rezept für Gabi's Nusslebkuchen. Es eignet sich zum Backen, vorausgesetzt, man hat alle Zutaten und einen Backofen mit Backblech. Mit dem Teig werden kleine Plätzchen auf Oblaten gesetzt und gebacken. Die fertigen Küchlein kann man in ein Museum bringen oder unter den Weihnachtsbaum stellen. Man kann sie auch einfach fressen.
+
+## Zutaten
+
+* 1 Teelöffel Zimt
+* 1 Messerspitze gemahlenen Kardamon
+* 70 g Schmalz
+* 150 g Zucker
+* 2 Eier
+* 1 Teelöffel Kakao
+* 100 g Zitronat
+* 100 g Orangeat
+* 200 g Nüsse oder Mandeln
+* 250 g Mehl
+* 1 Paket Backpulver
```

Comment:
Vielleicht ist für einige Backunkundige die Bezeichnung "_Paket_" Backpulver irreführend. Da sonst immer nur von Päckchen gesprochen wird und im normalen Supermarkt Backpulver auch nicht im 1 kg Paket verkauft wird. 

Ich schlage vor das "_Paket_" in "**Päckchen**" zu ändern.